### PR TITLE
Adding Atbash Cipher Program

### PR DESCRIPTION
**Pull request by Ihjass**

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thank you for submitting a PR! 😄 Please add your algorithm file in `src/TheAlgorithms.jl`. You can check other merged PRs such as in PR #103. Let us know if you need any help here or in our [**Discord server**](https://discord.gg/c7MnfGFGa6). 😃

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

File: src/cipher/atbash.jl [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  parts = []
+  part = ""
+  for r in input
+    part *= xform(r)
+    if length(part) >= 5
+      push!(parts, part)
+      part = ""
+    end
+  end
+  if part != ""
```
Comment (uncomfyhalomacro) [inline src/cipher/atbash.jl:58]:
```suggestion
  if !isempty(part)

```

**Review by uncomfyhalomacro — COMMENTED** [review]
@Ihjass Please check. Just a small change :slightly_smiling_face:

**Review by uncomfyhalomacro — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks for helping, @uncomfyhalomacro! 😄

**Pull request merged**